cli/tests/nonce.rs

```diff
@@ -122,7 +122,7 @@ fn full_battery_tests(
     // Create nonce account
     config_payer.command = CliCommand::CreateNonceAccount {
         nonce_account: read_keypair_file(&nonce_keypair_file).unwrap().into(),
-        nonce_authority: read_keypair_file(&authority_keypair_file).unwrap().pubkey(),
+        nonce_authority: Some(read_keypair_file(&authority_keypair_file).unwrap().pubkey()),
```

Comment:
I think there should be integration tests for the implied nonce authority case. `nonce_authority: None`, etc.

Reply:
Hrm.  Yeah, I'll fix up the non-authority case to call through with `None` instead of the same keypair